js/researches/sentencesLength.js

```diff
@@ -0,0 +1,18 @@
+var wordCount = require( "../stringProcessing/countWords.js" );
+var forEach = require( "lodash/forEach" );
+
+/**
+ * Returns the number of words in a sentence.
+ * @param {Array} sentences array with sentences from text.
+ * @returns {Array} array with amount of words in each sentence.
+ */
+module.exports = function( sentences ) {
+	var sentencesCount = [];
```

Comment:
Would be nicer to have an `array.map` and `array.filter` for this functionality.